src/libs/ajax/AzureStorage.ts

```diff
@@ -148,7 +150,16 @@ export const AzureStorage = (signal?: AbortSignal) => ({
         const textFileContents = await getObject()
         return fetchOk(`${getConfig().calhounUrlRoot}/${calhounPath}`,
           _.mergeAll([authOpts(), { signal, method: 'POST', body: textFileContents }])
-        ).then(res => res.text())
+        ).then(res => {
+          //@ts-expect-error
+          Ajax().Metrics.captureEvent(Events.analysisPreviewSuccess, { fileName: blobName, fileType: getAnalysisFileExtension(blobName), cloudProvider: cloudProviderTypes.AZURE })
```

Comment:
Suggest `cloudPlatform` as the key for consistency with other Metrics.

https://github.com/DataBiosphere/terra-ui/blob/8faa57e81ba5f87fe890282571794e579cb20049/src/libs/events.js#L110-L142